README.md

```diff
@@ -1,7 +1,95 @@
 # BestFlow
 We are going to fix bad chatbots!
 
-Hosted at: https://bestflow.netlify.app/
+Diagnostic tool for chatbot transcripts that help retailers identify poor or strong questions and intents through an easily digestible and intuitive display.
 
-This project uses:
-React, Axios, Node, Tailwind, MongoDB, Express and more :D
+We aim to:
+- Reduce time spent on chatbot optimization
+- Reduce bad chatbot intents
+- Display an average of 30% of diagnostic problems in any transcripts’ flow
+- Optimize the flow of at least three intents in x transcripts
+
+Each transcript will be displayed with a question and the responses it leads to.
+Each response also has a percentage displayed to show the chances of leading to a specific branch.
+Our helpful star tool will also let users tag which questions they think are important to add to a chatbot.
+They can also use the flag tool to tag questions that are not helpful and need to be rewritten or deleted.
```

Comment:
Is there value in having emojis here. Is that unprofessional? just a thought

I think there are star and red flag emojis at least. I have definitely seen the red flag one...

Reply:
I agree on adding emojis

It is slightly informal, but not in a bad way for a school project/personal project